server/asset/dcr/dcr.go

```diff
@@ -686,3 +691,10 @@ func toCoinID(txHash *chainhash.Hash, vout uint32) []byte {
 func toAtoms(v float64) uint64 {
 	return uint64(math.Round(v * 1e8))
 }
+
+// isTxNotFoundErr will return true if the error indicates that the requested
+// transaction is not known.
+func isTxNotFoundErr(err error) bool {
+	var rpcErr *dcrjson.RPCError
+	return errors.As(err, &rpcErr) && rpcErr.Code == dcrjson.ErrRPCInvalidAddressOrKey
```

Comment:
I think we want `dcrjson/v3` imported since rpcclient uses that version...  Actually, it might be what dcrd's rpc server is build with that matters.   Hmm, if the code is whatever integer comes from the node, this could be fragile, but I don't have a better suggestion.